app/views/dashboards/search.js.erb

```diff
@@ -1,6 +1,6 @@
 $('#search_results_shell').empty();
-<% if @results.blank? %>
-  $("#search_results_shell").append("<%= escape_javascript(render partial: 'patients/new_patient', locals: { patient: @patient, name: @name, phone: @phone, today: @today }) %>");
-<% else %>
+$("#search_results_shell").append("<%= escape_javascript(render partial: 'patients/new_patient', locals: { patient: @patient, name: @name, phone: @phone, today: @today }) %>");
+<% if @results.any? %>
+  $("#search_results_shell").append("<div class='row m-t'></div>");
```

Comment:
hmm, that this is needed is probably indicative of a problem elsewhere -- I'll run a quick fixer upper.